ansible/plugins/filter/filters.py

```diff
@@ -21,9 +21,10 @@
 # IN THE SOFTWARE.
 #
 
-from ansible.errors import AnsibleFilterError
 import re
 
+from ansible.errors import AnsibleFilterError
```

Comment:
Does this line need to be moved?

Reply:
IMHO it does not "need" to be moved, but this is good [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) style, so I'm +1.

Reply:
It for sure doesn't "need" to be moved but as @refack said it's just about following PEP 8 style guide lines.